include/aws/sdkutils/aws_profile.h

```diff
@@ -99,12 +99,23 @@ const struct aws_profile *aws_profile_collection_get_profile(
     const struct aws_profile_collection *profile_collection,
     const struct aws_string *profile_name);
 
+AWS_SDKUTILS_API
+const struct aws_profile *aws_profile_collection_get_sso_session(
+    const struct aws_profile_collection *profile_collection,
+    const struct aws_string *profile_name);
+
 /**
- * Returns how many profiles a collection holds
+ * Returns the number of profiles in a collection
  */
 AWS_SDKUTILS_API
 size_t aws_profile_collection_get_profile_count(const struct aws_profile_collection *profile_collection);
 
+/**
+ * Returns the number of sso-sessions in a collection
+ */
+AWS_SDKUTILS_API
+size_t aws_profile_collection_get_sso_session_count(const struct aws_profile_collection *profile_collection);
```

Comment:
should we have a generic api that can return map of all values for a given section type and name?

Reply:
Thanks, fixed.